### PR TITLE
Update cli flags 2.4

### PR DESCRIPTION
**Pull request by Swiftb0y**
manual PR for https://github.com/mixxxdj/mixxx/pull/11474/ along with some housekeeping.

### CODE REVIEW

File: source/chapters/appendix/commandline_dev_tools.rst [hunk 1]
```diff
@@ -40,50 +40,58 @@ types, go to :ref:`file-format-compatibility`.
...
-                        ``LEVEL`` is one of the values defined at ``--logLevel``
-                        above.
---debugAssertBreak      Breaks (``SIGINT``) Mixxx if a ``DEBUG_ASSERT`` evaluates
-                        to false. A debugger can then be used to continue.
-                        This disables the ``MIXXX_DEBUG_ASSERTIONS_FATAL``
-                        flag which can otherwise lead to a time consuming
-                        full rebuild.
--h, --help              Display this help message and exit
+--resource-path PATH            Top-level directory where Mixxx will look for
+                                its resource files such as MIDI mappings,
```
Comment (ronso0) [inline source/chapters/appendix/commandline_dev_tools.rst:44]:
```suggestion
                                its resource files such as skins and MIDI mappings,
```

Comment (Swiftb0y) [reply source/chapters/appendix/commandline_dev_tools.rst:44]: These are taken straight from mixxx, so we'd need to change the text there as well for consistency sake.

**Review by JoergAtGithub — APPROVED** [review]
LGTM! Does everybody agrees to merge this?

**Comment by Swiftb0y** [comment]
thank you, rebased and force-pushed to fix CI.

**Comment by Swiftb0y** [comment]
yey, all green.

**Review by ronso0 — APPROVED** [review]

**Comment by JoergAtGithub** [comment]
Thank you!

**Comment by Swiftb0y** [comment]
Thank you.

**Pull request merged**